assets/javascripts/discourse/components/minimal-gamification-leaderboard.js

```diff
@@ -0,0 +1,65 @@
+import Component from "@ember/component";
+import discourseComputed from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
+import LoadMore from "discourse/mixins/load-more";
+import { ajax } from "discourse/lib/ajax";
+import { popupAjaxError } from "discourse/lib/ajax-error";
+
+export default Component.extend(LoadMore, {
+  tagName: "",
+  eyelineSelector: ".user",
+  page: 1,
+  loading: false,
+  canLoadMore: true,
+  period: "all",
+  notTop10: true,
+
+  init() {
+    this._super(...arguments);
+
+    ajax(`/leaderboard`)
+      .then((model) => {
+        this.set("model", model);
+      })
+      .catch(() => this.replaceWith("/404"));
```

Comment:
Hmm... may be best to just have this fail silently, since it's very much an optional component.